stylesheets/components/CompositionArea.scss

```diff
@@ -55,6 +55,8 @@
     align-items: center;
     background: none;
     border: none;
+    width: 32px;
```

Comment:
It is a good practice to solve one problem per PR ( because if something goes wrong it can be reverted very easily ). Can you make another PR just for that?